src/net/sourceforge/kolmafia/request/FightRequest.java

```diff
@@ -851,6 +854,19 @@ public static final boolean canRockFlyer() {
         && !KoLCharacter.inGLover();
   }
 
+  public static int getCurrentRAM() {
+    // RAM starts out at a particular value and decrements through the
+    // fight as cyber skills are used.
+    return (currentRound == 0) ? calculateInitialRAM() : currentRAM;
+  }
+
+  public static int calculateInitialRAM() {
+    return 3
```

Comment:
`3 + KoLCharacter.currentNumericModifier(DoubleModifier.RAM)` should do it I think

Reply:
It needs a cast, but otherwise, right. Done.